### PR TITLE
support changing keyfile

### PR DESCRIPTION
**Pull request by Evidlo**

### CODE REVIEW

**Comment by Evidlo** [comment]
Strange.  I can't reproduce this failure.

File: tests/tests.py [hunk 1]
```diff
@@ -214,21 +214,21 @@ def test_fields(self):
...
-        shutil.copy(base_dir + '/test.kdbx', base_dir + '/change_pass.kdbx')
+        shutil.copy(base_dir + '/test.kdbx', base_dir + '/change_creds.kdbx')
         self.kp = pykeepass.PyKeePass(base_dir + '/test.kdbx', password='passw0rd', keyfile=base_dir + '/test.key')
-        self.kp_pass = pykeepass.PyKeePass(base_dir + '/change_pass.kdbx', password='passw0rd', keyfile=base_dir + '/test.key')
+        self.kp_pass = pykeepass.PyKeePass(base_dir + '/change_creds.kdbx', password='passw0rd', keyfile=base_dir + '/test.key')
 
-    def test_set_password(self):
-        self.kp_pass.set_password('f00bar')
+    def test_set_credentials(self):
+        self.kp_pass.set_credentials(password='f00bar', keyfile='/tmp/change.key')
```
Comment (pschmitt) [inline tests/tests.py]: `/tmp/change.key`. That should be `base_dir + '/change.key'`

**Comment by pschmitt** [comment]
Well I can. `/tmp/change.key` points to something outside this repository.

**Comment by pschmitt** [comment]
okay thanks. 
In the future we should probable be using `os.path.join` instead of appending strings in regards of files. This way tests should pass on OSes other than linux/bsd.

**Comment by Evidlo** [comment]
Could you bump the version, please?

**Comment by pschmitt** [comment]
Yep. Will do.

**Comment by pschmitt** [comment]
Done:

- https://github.com/pschmitt/pykeepass/releases/tag/2.7.2
- https://pypi.python.org/pypi/pykeepass/2.7.2

**Pull request merged**